message/message-util/util/src/java/org/sakaiproject/message/util/BaseMessage.java

```diff
@@ -1186,9 +1185,9 @@ protected boolean isIntersectionGroupRefsToGroups(Collection groupRefs, Collecti
 		return rv;
 	}
 	
-	private boolean isUserInstructor(String userId, Site site){
+	private boolean canSeeAllGroups(String userId, Site site){
```

Comment:
Thanks the name change is good, do we still think BaseAnnouncementService needs to override this to allow for annc.all.groups? Maybe just change this to protected in case.